Formula/bd.rb

```diff
@@ -0,0 +1,18 @@
+class Bd < Formula
+  desc "Quickly go back to a parent directory in linux"
+  homepage "https://github.com/vigneshwaranr/bd"
+  url "https://github.com/vigneshwaranr/bd/archive/v1.02.tar.gz"
+  sha256 "66133d21cfa2bffae4ddfc34060635646d7c7e21983e91818e18d7521163bd91"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    bin.install "bd"
+    bash_completion.install "bash_completion.d/bd"
+  end
+
+  test do
+    assert_match "Version: 1.02", shell_output("bd -v")
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula